lib/htmllint.js

```diff
@@ -39,8 +54,13 @@ module.exports = function( config, done ) {
     var files = config.files.map( path.normalize );
     async.mapSeries( chunkify( files, maxChars ), function( chunk, cb ) {
 
+      // determine proper jarfile invokation and arguments.
```

Comment:
typo: invokation/invocation. 

Since its the cmd for exec, how about `// determine proper jarfile command and arguments`?